stdlib/2and3/xml/etree/ElementTree.pyi

```diff
@@ -10,53 +10,80 @@ class ParseError(SyntaxError): ...
 
 def iselement(element: 'Element') -> bool: ...
 
-_Ss = TypeVar('_Ss', str, bytes)
 _T = TypeVar('_T')
-_str_or_bytes = Union[str, bytes]
+
+# Type for parser inputs. Parser will accept any unicode/str/bytes and coerce,
+# and this is true in py2 and py3 (even fromstringlist() in python3 can be
+# called with a heterogeneous list)
+_parser_input_type = Union[bytes, Text]
+
+# Type for individual tag/attr/ns/text values in args to most functions.
+# In py2, the library accepts str or unicode everywhere and coerces
+# aggressively.
+# In py3, bytes is not coerced to str and so use of bytes is probably an error,
+# so we exclude it. (why? the parser never produces bytes when it parses XML,
+# so e.g., element.get(b'name') will always return None for parsed XML, even if
+# there is a 'name' attribute.)
+_str_argument_type = Union[str, Text]
+
+# Type for return values from individual tag/attr/text values and serialization
+if sys.version_info >= (3,):
+    # note: in python3, everything comes out as str, yay:
+    _str_result_type = str
+    # unfortunately, tostring and tostringlist can return either bytes or str
+    # depending on the value of `encoding` parameter. Client code knows best:
+    _tostring_result_type = Any
```

Comment:
Yeah, this is pretty sad. The docs say:
> Use encoding="unicode" to generate a Unicode string (otherwise, a bytestring is generated).
The default value for the `encoding` argument is "us-ascii". So on Python 3 this was wrong all along (most of the time).